javascript/react/src/components/HistoryPanel.styles.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import styled from 'styled-components'
+
+export const icons = {
+	cancel: { iconName: 'Cancel' },
+}
+
+export const Aside = styled.div<{ isCollapsed: boolean }>`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	height: 100%;
+	overflow: hidden;
+	box-shadow: ${({ isCollapsed }) =>
+		isCollapsed ? 'none' : '-1px 0px 10px 0px rgba(0,0,0,0.3)'};
```

Comment:
Don't use hard-coded colors: pull in the `theme` object (which should be injected in context and available as an arg automatically, see other styles below).